apps/openmw/mwgui/tradeitemmodel.cpp

```diff
@@ -24,6 +24,18 @@ namespace MWGui
         return mItems[index];
     }
 
+    ItemModel::ModelIndex TradeItemModel::getIndex(ItemStack item)
+    {
+        size_t i = 0;
+        for (std::vector<ItemStack>::iterator it = mItems.begin(); it != mItems.end(); ++it)
+        {
+            if (*it == item)
+                return i;
+            ++i;
+        }
+        return -1;
```

Comment:
This method was just copied over from all the other getIndex implementations around in the codebase. 
Would probably better to fix all these in a seperate PR (if it is even agreed that it is better) and fix all the code duplication while at it